docs/reference.rst

```diff
@@ -98,6 +99,8 @@ Example configuration:
   be stored (e.g. slot specific metadata, see :ref:`slot-status`).
   This file should be located on a filesystem which is not overwritten during
   updates.
+  This method is preferable over per-slots status files as it allows to store
```

Comment:
To avoid having a recurrence at the beginning of the sentence compared to the previous one I propose to start it like

```
A central status file is preferable…
```

instead. While at it, please do ``s/preferable over per-slots/preferable to per-slots/`` and ``s/per-slots/per-slot/`` as well.

Reply:
Do you know about GH suggestions? ;) I'll change it as you requested.

Reply:
> Do you know about GH suggestions? ;) I'll change it as you requested.

Having a joint coffee together in the office later will offer a good possibility for introducing me to GH suggestions! ;-)